test_only_uses_tls: failure on Fedora with 1.10.4

So I was updating docker-py in Fedora land to `1.10.4` and this happened during the RPM build process:

```
tests/unit/ssladapter_test.py:27: in test_only_uses_tls
    assert ssl_context.options & OP_NO_SSLv2
E   AssertionError: assert (2181170175L & 0)
E    +  where 2181170175L = <ssl.SSLContext object at 0x3ffa4cbdde8>.options
```

https://kojipkgs.fedoraproject.org//work/tasks/2344/16112344/build.log

Since we unbundle urllib3 from requests it could be the issue. Or some system default changed. Didn't have time to dig through.

Creating the issue just to see if any other distros bumped into this.

--- a/tests/unit/ssladapter_test.py
+++ b/tests/unit/ssladapter_test.py
@@ -24,7 +24,8 @@ class SSLAdapterTest(base.BaseTestCase):
         ssl_context = ssladapter.urllib3.util.ssl_.create_urllib3_context()
 
         assert ssl_context.options & OP_NO_SSLv3
-        assert ssl_context.options & OP_NO_SSLv2
+        # if OpenSSL is compiled without SSL2 support, OP_NO_SSLv2 will be 0
+        assert not bool(OP_NO_SSLv2) or ssl_context.options & OP_NO_SSLv2
         assert not ssl_context.options & OP_NO_TLSv1
 
 

